docs/shortcuts.md

```diff
@@ -5,16 +5,25 @@ There are a number of shortcuts supported for frequently used actions.
 |                               Shortcut                               | Description                                          |
 | :------------------------------------------------------------------: | ---------------------------------------------------- |
 |                             *Navigation*                             |                                                      |
-|                          Left Mouse Button                           | Rotates the Point Cloud                              |
-|                          Right Mouse Button                          | Translates the Point Cloud                           |
+|                          Left Mouse Button                           | Rotates the camera around Point Cloud centroid       |
+|                          Right Mouse Button                          | Translates the camera                                |
 |                             Mouse Wheel                              | Zooms into the Point Cloud                           |
 |                             *Correction*                             |                                                      |
-|         `W`, `A`, `S`, `D` <br> `Ctrl` + Right Mouse Button          | Translates the Bounding Box back, left, front, right |
+|                          `W`, `A`, `S`, `D`                          | Translates the Bounding Box back, left, front, right |
+|                     `Ctrl` + Right Mouse Button                      | Translates the Bounding Box in all dimensions        |
 |                               `Q`, `E`                               | Lifts the Bounding Box up, down                      |
-|                               `X`, `Y`                               | Rotates the Boundign Box around z-Axis               |
+|                               `Z`, `X`                               | Rotates the Bounding Box around z-Axis               |
```

Comment:
I am a bit confused here. I thought this file is just for generating the [interactive documentation](https://ch-sa.github.io/labelCloud/shortcuts/) and should be the same as https://github.com/ch-sa/labelCloud/blob/master/README.md

All the changes in this file are from copying [these lines](https://github.com/ch-sa/labelCloud/blob/master/README.md?plain=1#L108-L130) into the `shortcuts.md` file, as I thought it was outdated. I have not done any changes shortcut changes apart from the `,` and `.`.

It would be great if you could point me to the misunderstanding. If this file should not be the same as `README.md` I will simply revert the changes here

Reply:
Yes, you're right. It is already `X` + `Z` for rotating around the Z-axis.